utils/find-overlay-deps-closure.sh

```diff
@@ -39,9 +41,10 @@ function find_deps() {
 
     local REGEX="./Frameworks/(${OVERLAY_NAME_ALTERNATION}).framework/.*|.*/usr/include/(xpc|dispatch|os|objc|simd)/.*\.h"
 
-    IFS='\
+    IFS='
```

Comment:
This changes the behavior and makes it take twice as long.

Put this inside the line 48 loop and see how it calls the regex on ``\`` lines as well. 
```            echo "line: $line" >&2  # line 49```